docs/manual/Makefile

```diff
@@ -71,7 +71,7 @@ manual.html: manual.pdf CFLogo.png favicon-checkerframework.png ../api
 	sed -i -e "s%<style type=\"text/css\">%<style type=\"text/css\">\nimg { max-width: 100\%; max-height: 100\%; }%" manual.html
 
 ../api:
-	cd .. && ln -s ../checker/api api
+	cd .. && ln -fs ../checker/api api
```

Comment:
I put a note in my commit message about this. The rule will _also_ run if the symlink `../api` exists but points to a folder that does not exist. That can happen if you have never built `../../checker/api`. The "right" fix here might be for the Makefile to build `../../checker/api` and its contents, but I wasn't confident I could get that working.